widget/embed_ox_talks.js

```diff
@@ -194,5 +303,39 @@ var oxtalks = {
             selector = this.createContainer();
         }
         this.queryTalks(params, this.buildList, $(selector));
-    }
+    },
+
+    //Create a calendar using fullCalendar. The query will be invoked by fullCalendar
+    showCalendar: function(params, selector) {
+        //append to body if no selector specified
+        if (!selector) {
+            selector = this.createContainer();
+        }
+        
+        //check that the fullcalendar plugin is present and provide helpful feedback if not
+        if(!$.fullCalendar)
+        {
+            console.error("Fullcalendar plugin not present. Please consult the docs at http://talksox.readthedocs.org/en/latest/");
+            //error message for the end user
+            $(selector).text("Unable to display calendar");
```

Comment:
Maybe provide better error message if people don't look at the console?
`Unable to display calendar / missing full calendar plugin`